pysand/fluidproperties.py

```diff
@@ -104,3 +104,40 @@ def mix_viscosity(P, T, Qo, Qw, Qg, mu_o, mu_w, mu_g, Z):
     # Calculations
     mu_m = (mu_o + Qw/Qo * mu_w + Qg/Qo*P0*T*Z / (P * T0) * mu_g) / (1 + Qw/Qo + Qg/Qo*P0*T*Z / (P * T0))
     return np.round(mu_m, 6)
+
+
+def liq_density(Qo, Qw, rho_o, rho_w):
+    """
```

Comment:
add in docstring what the function does, something like "Simple weighted average of liquid density"